source/particle/world.cc

```diff
@@ -354,6 +365,20 @@ namespace aspect
       unsigned int n_particles_in_cell = 0;
       switch (status)
         {
+#if DEAL_II_VERSION_GTE(9,6,0)
+          case CellStatus::cell_will_persist:
+          case CellStatus::cell_will_be_refined:
+            n_particles_in_cell = particle_handler->n_particles_in_cell(cell);
+            break;
+
+          case CellStatus::cell_invalid:
+            break;
+
+          case CellStatus::children_will_be_coarsened:
+            for (const auto &child : cell->child_iterators())
+              n_particles_in_cell += particle_handler->n_particles_in_cell(child);
+            break;
```

Comment:
Same here. I don't think it would hurt to make the names available again via using declarations or similar trickery.